openstackgit-update.yml

```diff
@@ -58,7 +49,31 @@
       with_items: "{{ local_packages.results.0.item.remote_package_parts }}"
       register: git_clone
       until: git_clone | success
-      retries: 2
+      retries: 5
+      delay: 5
+
+  vars:
+    working_dir: "{{ lookup('ENV', 'PWD') }}"
+
+- name: Push the updated clones to rpc-repo
+  hosts: mirrors
+  tasks:
+
+    - name: Ensure the openstackgit folder is present
+      file:
+        path: "/var/www/repo/openstackgit"
+        state: directory
+
+    - name: Synchronize cache
+      synchronize:
+        src: "{{ working_dir }}/openstackgit"
+        dest: "/var/www/repo/openstackgit"
```

Comment:
Not sure myself - I was just going to kinda see what happens and adjust afterwards.